data/data/manifests/bootkube/09-openshift-service-ca-operator-namespace.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  # This is the namespace used to hold the service-ca operator.
+  name: openshift-service-ca-operator
```

Comment:
Per [group arch call](https://docs.google.com/a/redhat.com/document/d/1Mvs1emDevbCNl3xcXn0igWJGt_9a4DziwQV7y9-VWUI/edit?disco=AAAACfWeAX0), I think we want to be using `openshift-service-ca` (dropping the `-operator` suffix)

Reply:
I'm using the same pattern as the other operators, eg.
operator namespace: openshift-authentication-operator
operand namespace: openshift-authentication
So here we have
operator namespace: openshift-service-ca-operator
operand namespace: openshift-service-ca (would be changed from openshift-service-cert-signer).

My intent is to stage this for the operator to then use a sync controller to sync to openshift-service-ca for the operand controllers to use. This way, we will be able to control the operand  namespace with the operator instead of needing another PR to the installer to change the namespace.